web/pandas_web.py

```diff
@@ -441,19 +484,28 @@ def main(
     For ``.md`` and ``.html`` files, render them with the context
     before copying them. ``.md`` files are transformed to HTML.
     """
-    config_fname = os.path.join(source_path, "config.yml")
-
     shutil.rmtree(target_path, ignore_errors=True)
     os.makedirs(target_path, exist_ok=True)
 
     sys.stderr.write("Generating context...\n")
-    context = get_context(config_fname, target_path=target_path)
+    context = get_context(
+        os.path.join(source_path, "config.yml"),
+        navbar_fname="navbar.yml",
```

Comment:
If you don't want to hardcode it where it's used, maybe better to add the name of the navbar file to the config. Either way is good, but specifying it here seems a bit arbitrary and not the place where we'd look for it given how the code is structured.

Reply:
Fixed.